cdci_data_analysis/analysis/parameters.py

```diff
@@ -431,30 +435,30 @@ def value(self):
     @value.setter
     def value(self, v):
         if v is not None and v!='':
-            self.check_int_value(v,name=self.name)
+            self.check_int_value(v, name=self.name)
             self._v = np.int(v)
 
         else:
             self._v=None
 
+    def get_value_in_default_format(self):
+        self.check_int_value(self.value, name=self.name)
+        return int(self.value)
 
     @staticmethod
     def check_int_value(value, units=None,name=None):
         #print('check type of ',name,'value', value, 'type',type(value))
         if value is None or value=='':
             pass
         else:
+            if isinstance(value, float):
+                message = '%s is an invalid value for %s since it cannot be used as an Integer' % (value, name)
```

Comment:
hm, just curious, why did you moved away from f-strings? If anything, this '%' form is the oldest. f-string and `.format` are at least equivalent. 
See [note](https://realpython.com/python-string-formatting/) [pic](https://files.realpython.com/media/python-string-formatting-flowchart.4ecf0148fd87.png)

Reply:
I remember asking you the same question on another topic and you replied to me that, given the amount of logs produced, this more old-style formatting style would be less heavy, in comparison with the f-string one.

Reply:
this comment:

[https://github.com/oda-hub/dispatcher-plugin-integral/pull/65#discussion_r723526817](url)

Reply:
ha, well I guess I was not clear enough. Maybe the confusion was that I said logging (generic logging) and not `logging` (python module). That case applied to python module logging, used in the commented line. It is also what is referenced in the article linked in that comment.

If you re-read the comment with the `logging` module case, you realize why it matters.
What `logging` does is peculiar. In fact, it does not really use old-style formatting. It just uses very similar string,

For example, compare the next two lines:
```
logger.warning("unknown dev OSA version %s, allowed %s", bla, blabla)
logger.warning("unknown dev OSA version %s, allowed %s" % bla, blabla)
```

First line passes arguments to a logging.warning function, and function does something internally. What it does is somewhat intelligent and optimized.

Second line does the python string formatting, independently of the output levels, and then passes the whole string to `logging.warning`.

Reply:
ok is a bit more clear now, but yeah there are several cases to take into account, and that got me a bit confused especially with the various terminology

whereas this specif case is related to a simple string formatting where for us the `f-{}` is the suggested option

Reply:
It's hard not to get confused with python string formatting. There are like 5 different ways, some look similar (for "convenience") but are in fact quite different.